apps/coa/behaviours/save-document.js

```diff
@@ -13,55 +13,83 @@ module.exports = (documentCategory, name) => superclass => class extends supercl
   }
 
   validateField(key, req) {
-    if (req.body['upload-file']) {
-      const fileUpload = req.files[name];
-
-      if (fileUpload) {
-        const uploadSize = fileUpload.size;
-        const mimetype = fileUpload.mimetype;
-        const uploadSizeTooBig = uploadSize > config.upload.maxFileSizeInBytes;
-        const uploadSizeBeyondServerLimits = uploadSize === null;
-        const invalidMimetype = !config.upload.allowedMimeTypes.includes(mimetype);
-        const invalidSize = uploadSizeTooBig || uploadSizeBeyondServerLimits;
-
-        if (invalidSize || invalidMimetype) {
-          return new this.ValidationError('document-file', {
-            type: invalidSize ? 'maxFileSize' : 'fileType',
-            redirect: undefined
-          });
-        }
-      } else {
-        return new this.ValidationError('document-file', {
-          type: 'required',
-          redirect: undefined
-        });
+    const fileToBeValidated = req.files[name];
+    const documentsByCategory = req.sessionModel.get(documentCategory) || [];
+    const validationErrorFunc = type => new this.ValidationError(key, { type: type });
+
+    // When click Continue without uploading a file.
+    if (req.body.Continue && documentsByCategory.length === 0) {
```

Comment:
It might be better to rename "Continue" to "continueWithoutUpload" to avoid confusion with the generic "Continue" behaviour.

Also in case there will be a requirement for an optional upload, my suggestion would be to do a limit check as well

Reply:
This is the continue button in the page which, user will do continue with or without file upload. so i have kept it as continue.